spec/controllers/job_definitions_controller_spec.rb

```diff
@@ -38,6 +38,68 @@
 
       expect(assigns(:definition)).to be_new_record
     end
+
+    context 'with dup_from params' do
+      subject! { get :new, params: { dup_from: definition.id } }
+      it do
+        expect(response).to have_http_status(:ok)
+        expect(response).to render_template('new')
+
+        expect(assigns(:definition)).to be_new_record
+        expect(assigns(:definition).name).to eq definition.name
+        expect(assigns(:definition).admins).to eq definition.admins + [controller.current_user]
+        expect(assigns(:definition).tags).to eq definition.tags
+      end
+
+      context 'with current_user and aother admin user' do
```

Comment:
[typo] another?

Reply:
Thanks, I fixed it in https://github.com/cookpad/kuroko2/pull/92/commits/33ff102f4903dff4accb77200681ae8bdd70de3f.